### ISSUE TITLE
compilation error: not found: value allCredentials

### ISSUE THREAD

**Issue by yanns** [fthomas/scala-steward#1473]
For one project, I have a compilation error when sbt is loading:

```
[error] /root/.sbt/1.0/plugins/StewardPlugin.scala:42:30: not found: value allCredentials
[error]         val sbtCredentials = allCredentials.value
[error]                              ^
[error] one error found
[error] (Compile / compileIncremental) Compilation failed
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore?
	at org.scalasteward.core.io.process$.$anonfun$slurp$10(process.scala:49)
```

We're using sbt `1.2.8`. Maybe that's the cause.

**Comment by yanns** [comment]
`allCredentials` is used since https://github.com/fthomas/scala-steward/pull/1460

**Comment by mzuehlke** [comment]
It seems that `allCredentials` has been added in sbt 1.3.0 (https://github.com/sbt/sbt/pull/4855)

**Comment by fthomas** [comment]
Oops, I should have noticed that `allCredentials` is not available in all sbt versions.

Maybe we could use the same trick as in https://github.com/fthomas/scala-steward/commit/f2e3c3daccf399240c2123bbb30855d7c682608c to call `allCredentials` if it is available and fall back to `credentials` otherwise.

**Comment by fthomas** [comment]
Fixed by #1480.
